UI/profile.html

```diff
@@ -30,9 +31,28 @@
 
 
                 <div class="profile-info">
-                    <div class="img-container">
-                            <img id="photo" src="images/samples/user1.png"/>
+                    <div class="profile-details">
+                        <details>
+
+                            <summary>
+                                    <img id="photo" src="images/user.svg"/>
+                                    <div class="img-container" id="img-cont">
+                                           
+                                            <div class="edit-image">
+                                                    <svg height="16" width="16" class="edit-icon" viewBox="0 0 14 16" version="1.1" aria-hidden="true"><path fill-rule="evenodd" d="M0 12v3h3l8-8-3-3-8 8zm3 2H1v-2h1v1h1v1zm10.3-9.3L12 6 9 3l1.3-1.3a.996.996 0 0 1 1.41 0l1.59 1.59c.39.39.39 1.02 0 1.41z"></path></svg>
+                                             Edit   
+                                            </div>
+                                    </div>
+                            </summary>
+                            <details-menu class="dropdown-menu">
+                              <label for="input-field">Upload an image</label>
+                              
+                              <button id="rm-btn">Remove image</button>
+                            </details-menu>
+                            <input type="file" id="input-field">
```

Comment:
As it currently is someone can even pick a video. The accept property can help in filtering out unwanted file types from the User's filesystem. Apart from this everything else works as you had intended it to. Great job :+1: 
```html
      <input accept="image/*"  type="file" />
```